canonical/ubuntu.com
# Refresh /landscape/install

## Done

- Refresh /landscape/install based on [copydoc](https://docs.google.com/document/d/1tU1z9PUm_kSwGJEsopJibE4gI8Gfv_3Dw_qN2_io_ao/edit#heading=h.yij8xaij9lcy)
- Removes unused styles

## QA

- Go to https://ubuntu-com-12675.demos.haus/landscape/install and check the content matches the [copydoc](https://docs.google.com/document/d/1tU1z9PUm_kSwGJEsopJibE4gI8Gfv_3Dw_qN2_io_ao/edit#heading=h.yij8xaij9lcy)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-2448
